src/api/spec/features/beta/webui/packages_spec.rb

```diff
@@ -92,6 +92,7 @@
       visit package_show_path(package: package, project: user.home_project)
       click_link('Requests')
       expect(page).to have_css('table#all_requests_table tbody tr', count: 1)
+      first('table#all_requests_table tbody tr td').click if mobile? # Expand datatable collapsed row
```

Comment:
This comment should be the commit message body...

Reply:
Moved to the commit message body.